File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/kotlin/ADV-06_R.kt
fun main() {
    println("ADV-START")
    System.out.flush()
    val a: Double = readln().trim().toDouble()
    val b: Double = readln().trim().toDouble()
    val m: Double = a / b
    val xs: DoubleArray = doubleArrayOf(3.0, m, 1.0)
    var best: Double = xs[0]
    for (i in 1 until xs.size) {
        val x: Double = xs[i]
        if (x > best) best = x
    }
    val selfeq: Boolean = m == m
    println("OBS=MAX:" + String.format("%.6f", best) + "|SELFEQ:" + selfeq)
    System.out.flush()
    println("ADV-END")
    System.out.flush()
}

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/python/ADV-06_R.py
print("ADV-START", flush=True)
a = float(input())
b = float(input())
m = a / b
xs = [3.0, m, 1.0]
best = xs[0]
for x in xs[1:]:
    if x > best:
        best = x
print("OBS=MAX:" + format(best, ".6f") + "|SELFEQ:" + str(m == m).lower(), flush=True)
print("ADV-END", flush=True)
